### PR TITLE
Show language selection to everyone

### PR DESCRIPTION
**Pull request by sethherr**
First, because the translator wants to see it. But also because why not!

- Setting language automatically leads to weird edge cases, and I don't think it adds much value. I've added a link to the edit account page, if there is a mismatch, which I think makes things better.
- When selecting a language, include the language's name in it's own language

### CODE REVIEW

**Comment by jmromer** [comment]
Does this break locale consistency on signup?

**Comment by sethherr** [comment]
@jmromer it did, good catch! Should be fixed by 6300a0f496d62a8d40df2b5afdd16fce06c2dc5b

File: spec/requests/locale_detection_request_spec.rb [hunk 1]
```diff
@@ -57,26 +50,29 @@
 
     context "given multiple detected locales" do
       include_context :request_spec_logged_in_as_user
-      before do
-        allow(Flipper).to receive(:enabled?).with(:localization, current_user).and_return(true)
-      end
 
       it "gives highest precedence to query param" do
-        current_user.update(preferred_language: :es)
+        current_user.update_attribute :preferred_language, :es
```
Comment (jmromer) [inline spec/requests/locale_detection_request_spec.rb]: A bit early to worry about this, but just a [head's up](https://github.com/rails/rails/pull/31998):

> DEPRECATION WARNING: update_attributes! is deprecated and will be removed from Rails 6.1 (please, use update! instead)

Comment (jmromer) [reply spec/requests/locale_detection_request_spec.rb]: Also, `update_attribute` (sans -_s_) bypasses validations, not clear if / why that's intended here.

Comment (sethherr) [reply spec/requests/locale_detection_request_spec.rb]: That is intended here. Because `:es` isn't an official language

**Pull request merged**